pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -17117,5 +17117,17 @@ final: prev:
     meta.homepage = "https://github.com/jhradilek/vim-snippets/";
   };
 
+  nvim-nio = buildVimPlugin {
```

Comment:
the script should have generated buildNeovimPlugin as nvim-nio exists in the lua package set. It's not either because it's not in master yet (in which case let's wait) or because you wrote this manually

Reply:
I added it manually.
So I removed the snippet and ran the script.